cairo-vm-cli/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 bincode = { version = "2.0.0-rc.2", tag = "v2.0.0-rc.2", git = "https://github.com/bincode-org/bincode.git" }
 cairo-vm = { path = "../vm", version = "0.6.0" }
+cairo-vm-tracer = { path = "../cairo-vm-tracer", version = "0.5.2" }
```

Comment:
We're going to use workspace dependencies moving forward. Could you move this to the root manifest and define it here as follows?
```
cairo-vm-tracer = { workspace = true, optional = true }
```

Reply:
Fixed